api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -267,12 +267,12 @@ message Bootstrap {
   // or any other DNS resolver types and the related parameters.
   // For example, an object of
   // :ref:`CaresDnsResolverConfig <envoy_v3_api_msg_extensions.network.dns_resolver.cares.v3.CaresDnsResolverConfig>`
-  // can be packed into this *typed_dns_resolver_config*. This configuration replaces the
+  // can be packed into this ``typed_dns_resolver_config``. This configuration replaces the
   // :ref:`dns_resolution_config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dns_resolution_config>`
   // configuration.
-  // During the transition period when both *dns_resolution_config* and *typed_dns_resolver_config* exists,
-  // when *typed_dns_resolver_config* is in place, Envoy will use it and ignore *dns_resolution_config*.
-  // When *typed_dns_resolver_config* is missing, the default behavior is in place.
+  // During the transition period when both ``dns_resolution_config`` and ``typed_dns_resolver_config`` exists,
+  // when ``typed_dns_resolver_config`` is in place, Envoy will use it and ignore ``dns_resolution_config``.
+  // When ``typed_dns_resolver_config`` is missing, the default behavior is in place.
```

Comment:
Is this covered by the RST warnings? Is there a style guide entry here?

Reply:
how could it - this is me editing prose to make words that should be literal, so - until we get AI in the pipeline this is not something we can test for

more generally - the use of `*` for literals relates to historic advice which i changed long ago

we could perhaps check for `*word*` and spellcheck the word as literals should not be marked up this way - it would catch some - ill look at it when i get to the spellcheck stuff

Reply:
I guess an alternative would be to clarify it in https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md, but as long as we don't detect this in tooling, it will be hard to catch.
Can we hook a validation to the comments parser somehow?

Reply:
this is what we do now for single backticks

for `*` or bare literals its much harder to detect - there are genuine uses of `*` - ie for italics/emphasis - and there is nothing to grep for with bare literals

spellchecking could work and catch most issues but not all and with some false +ves etc

this is something i intend to work, but not imminently - some preparatory work has been done over in pytooling tho

wrt to the STYLE guide i can update/review if you think it needs it but would suggest we follow up with that

Reply:
also - mho is that the best style guide is what is there - the more we clean stuff up the more people are working from examples that exemplify what we want

Reply:
> also - mho is that the best style guide is what is there - the more we clean stuff up the more people are working from examples that exemplify what we want

I also tend to follow other examples.